source/elements/oneDAL/source/index.rst

```diff
@@ -31,3 +31,4 @@ For general information, visit |github|.
    data_management/index.rst
    algorithms/index.rst
    bibliography.rst
+   terminology.rst
```

Comment:
Terminology should be in the beginning after introduction

Reply:
fixed